boa/src/value/mod.rs

```diff
@@ -392,7 +391,8 @@ impl JsValue {
         let _timer = BoaProfiler::global().start_event("Value::get_property", "value");
         match self {
             Self::Object(ref object) => {
-                let property = object.__get_own_property__(&key);
+                // TODO: had to skip `__get_own_properties__` since we don't have context here
```

Comment:
We could add the context as a parameter, right?

Reply:
I'll try to solve the todos

Reply:
Same as the previous one. It's a pretty big refactor. Maybe I'll tackle it after I finish merging this PR.